codeclimate/systems.yaml

```diff
@@ -67,7 +67,7 @@ systems:
           system: codeclimate
           function: api
         parameters:
-          URL: '{{ .sysData.url }}/orgs/{{ .ctx.org }}/repos'
+          request_path: 'orgs/{{ .ctx.org_id }}/repos'
```

Comment:
You won't be able to use parameters as `.ctx....`, so probably the older way is better.